aws/resource_aws_launch_configuration.go

```diff
@@ -247,6 +247,39 @@ func resourceAwsLaunchConfiguration() *schema.Resource {
 				},
 			},
 
+			"metadata_options": {
```

Comment:
This new configuration block and its arguments are missing documentation in `website/docs/r/launch_configuration.html.markdown`